sdk/src/Core/Amazon.Runtime/Internal/Auth/AWS4Signer.cs

```diff
@@ -663,12 +662,6 @@ public static string DetermineSigningRegion(IClientConfig clientConfig,
             // client config properties.
             if (alternateEndpoint != null)
             {
-#pragma warning disable CS0612,CS0618
-                var serviceEndpoint = alternateEndpoint.GetEndpointForService(serviceName, clientConfig.ToGetEndpointForServiceOptions());
-#pragma warning restore CS0612,CS0618
-                if (serviceEndpoint.AuthRegion != null)
```

Comment:
It was safe to remove the `serviceEndpoint.AuthRegion != null` check because the endpoint returned from `GetEndpointForService` sets the auth region to null.